### PR TITLE
JLovejoy_FinalProject

### PR DESCRIPTION
**Pull request by jalovejoy**
Prototype for social structure (work related project..)

### CODE REVIEW

**Comment by jalovejoy** [comment]
Here's the BitBalloon for it: http://clown-panda-81038.bitballoon.com/

Warning: code might have some unused Classes / Notes in it. I never had a chance to clean it up afterward.

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,195 @@
+* {
+	padding: 0;
+	margin: 0;
+	border: none;
+}
+
+body {
+	max-width: 100%;
```
Comment (Josephjbliss) [inline css/style.css:8]: This is unecessary as the max-width is already 100%

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+	display: block;
+	content: " ";
+	clear:both;
+}
+
+nav {
+	background-color: black;
+	width: 100%;
+	height: 40px;
+	position: absolute;
```
Comment (Josephjbliss) [inline css/style.css:24]: You should add a top and left position to this (best practice)

File: js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+	}
+};
+
+
+$(".cmi, .item, .miniItem").hover(function() {
+    $(this).children("ul").toggleClass("textOpen");
+});
+
+$(".marketing, .miniItem").hover(function() {
+	$(".miniItem").children("span").toggle("display", $("miniItem").css("display") === "block");
```
Comment (Josephjbliss) [inline js/scripts.js:89]: These should be `;` in stead of `,`

File: js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    $(this).children(".tooltipRightDown").css({top: (y + 10) + "px"});
+    $(this).children(".tooltipRightDown").css({left: (x + 10) + "px"});
+    $(this).children(".tooltipLeftUp").css({top: (y - 95) + "px"});
+    $(this).children(".tooltipLeftUp").css({left: (x - 215) + "px"});
+    $(this).children(".tooltipLeftDown").css({top: (y + 10) + "px"});
+    $(this).children(".tooltipLeftDown").css({left: (x - 210) + "px"});
+});
+
+$(".seo, .overlayEsc").click(function() {
+	$(".overlay").toggle("display"), $(".overlay").css("display") === "block);"
```
Comment (Josephjbliss) [inline js/scripts.js:104]: Same as above.

**Pull request closed without merging**